JS/Lesson32/onder/my-app/src/App.jsx

```diff
@@ -4,21 +4,29 @@ import {BD_SEARCH_BASE_URL} from './urls.js';
 import {RecipeList} from './components/RecipeList';
 import {SearchBar} from './components/SearchBar';
```

Comment:
This import is missing:
import { RecipeDetail } from './components/RecipeDetail';